lib/game_collection.rb

```diff
@@ -0,0 +1,17 @@
+require 'csv'
+
+class GameCollection
+  attr_reader :games_list
+
+  def initialize(file_path)
+    @games_list = create_games(file_path)
+  end
+
+  def create_games(file_path)
```

Comment:
moved the csv reader to each collection class, the stat_tracker will pass the file_path to the game_collection here

Reply:
I like this setup. I think it will be easier to pull from